cmd/lncli/commands.go

```diff
@@ -373,11 +374,33 @@ func sendCoins(ctx *cli.Context) error {
 			"sweep all coins out of the wallet")
 	}
 
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+	minConfs := int32(ctx.Uint64("min_confs"))
+
+	// In case that the user has specified the sweepall flag, we'll
+	// calculate the amount to send based on the current wallet balance.
+	if ctx.Bool("sweepall") {
+		balanceResponse, err := client.WalletBalance(
+			ctxc, &lnrpc.WalletBalanceRequest{
+				MinConfs: minConfs,
+			},
+		)
+		if err != nil {
+			return fmt.Errorf(
+				"unable to retrieve wallet balance: %w", err,
+			)
+		}
+		displayAmt = balanceResponse.GetConfirmedBalance()
+	} else {
+		displayAmt = amt
```

Comment:
I think we can remove this else branch to improve readability.
```
// In case that the user has specified the sweepall flag, we'll
// calculate the amount to send based on the current wallet balance.
displayAmt = amt
if ctx.Bool("sweepall") {
	balanceResponse, err := client.WalletBalance(
		ctxc, &lnrpc.WalletBalanceRequest{
			MinConfs: minConfs,
		},
	)
	if err != nil {
		return fmt.Errorf(
			"unable to retrieve wallet balance: %w", err,
		)
	}
	displayAmt = balanceResponse.GetConfirmedBalance()
}
```

Reply:
Done 🏁